### PR TITLE
Japanese translations

### PR DESCRIPTION
**Pull request by Yamagami-Takuma**
@f-higashi  
Could you review this?

### CODE REVIEW

**Comment by cheld** [comment]
CC @maciaszczykm

**Comment by cheld** [comment]
Super, thanks

File: i18n/messages-ja.xtb [hunk 1]
```diff
@@ -1308,7 +1308,7 @@
   <translation id="4060565009520367123" key="MSG_REPLICASETLIST_REPLICASETLIST_0" desc="Title for graph card displaying CPU metric of replica sets.">CPU使用量</translation>
   <translation id="4492539501833209064" key="MSG_REPLICASETLIST_REPLICASETLIST_1" desc="Title for graph card displaying memory metric of replica sets.">メモリー使用量</translation>
   <translation id="4602542864257969760" key="MSG_REPLICASETLIST_REPLICASETLIST_1" desc="Title for graph card displaying memory metric of replica sets.">メモリー使用量の履歴</translation>
-  <translation id="5987073561559829067" key="MSG_REPLICASETLIST_REPLICASETLIST_2" desc="Help message detailing what is included in the memory usage">The memory usage includes the caches in the pods managed by these replica sets.</translation>
+  <translation id="5987073561559829067" key="MSG_REPLICASETLIST_REPLICASETLIST_2" desc="Help message detailing what is included in the memory usage">  メモリー使用量は、これらのレプリカセットに管理されたポッドのキャッシュも含みます。</translation>
```
Comment (f-higashi) [inline i18n/messages-ja.xtb]: Remove leading whitespace.

File: i18n/messages-ja.xtb [hunk 2]
```diff
@@ -1443,8 +1443,8 @@
   <translation id="5650624948356810172" key="MSG_RESOURCELIMIT_RESOURCELIMITS_6" desc="Resource Limits max limit/request ratio entry.">最大上限/リクエスト</translation>
   <translation id="323516129286855678" key="MSG_RESOURCELIMIT_RESOURCELIMITS_7" desc="Resource Limits max limit/request ratio entry.">最大上限/リクエスト</translation>
   <translation id="2045128203058673243" key="MSG_RESOURCEQUOTADETAIL_RESOURCEQUOTADETAILSTATUS_0" desc="Resource quota details status section resource name.">リソース名</translation>
-  <translation id="6733342110976047464" key="MSG_RESOURCEQUOTADETAIL_RESOURCEQUOTADETAILSTATUS_1" desc="Resource quota details status section hard.">Hard</translation>
-  <translation id="3725558341406139723" key="MSG_RESOURCEQUOTADETAIL_RESOURCEQUOTADETAILSTATUS_2" desc="Resource quota details status section used.">Used</translation>
+  <translation id="6733342110976047464" key="MSG_RESOURCEQUOTADETAIL_RESOURCEQUOTADETAILSTATUS_1" desc="Resource quota details status section hard.">ハードウェア</translation>
```
Comment (f-higashi) [inline i18n/messages-ja.xtb]: `ハードリミット` is better.

File: i18n/messages-ja.xtb [hunk 3]
```diff
@@ -1584,38 +1584,38 @@
...
+  <translation id="310605795325552510" key="MSG_STORAGECLASSLIST_CARD_1" desc="Label \'Storage Class\' which will appear in the persistent volume delete dialog opened from a storageclass card on the list page.">ストレージクラス</translation>
+  <translation id="2857396329294182623" key="MSG_STORAGE_CLASS_LIST_CREATED_AT_TOOLTIP" desc="Tooltip 'Created at [some date]' showing the exact creation time of persistent volume."><ph name="CREATION_DATE"/> に作成</translation>
+  <translation id="4455224191195228523" key="MSG_THIRDPARTYRESOURCEDETAIL_DETAIL_0" desc="Title for objects card zerostate in third party resource details page.">表示するものがありません</translation>
+  <translation id="1909818214822603215" key="MSG_THIRDPARTYRESOURCEDETAIL_DETAIL_1" desc="Text for objects card zerostate in third party resource details page.">現在サードパーティーリソースのインスタンスはありません。</translation>
+  <translation id="8096714270041558618" key="MSG_THIRDPARTYRESOURCEDETAIL_DETAIL_1" desc="Text for objects card zerostate in third party resource details page.">現在第三者のサードパーティーリソースのインスタンスはありません。</translation>
```
Comment (f-higashi) [inline i18n/messages-ja.xtb]: I think it is good to  remove `第三者`.

File: i18n/messages-ja.xtb (hunk 1 above)
Comment (Yamagami-Takuma) [inline i18n/messages-ja.xtb]: fixed.

File: i18n/messages-ja.xtb (hunk 2 above)
Comment (Yamagami-Takuma) [inline i18n/messages-ja.xtb]: fixed.

File: i18n/messages-ja.xtb (hunk 3 above)
Comment (Yamagami-Takuma) [inline i18n/messages-ja.xtb]: fixed.

**Comment by cheld** [comment]
@maciaszczykm could you please have a quick check why the CI fails?

**Comment by floreks** [comment]
@cheld Travis fails because of #1721. Unfortunately also integration tests are failing. I'll try to fix that ASAP then we can update other PRs and merge them.

**Comment by maciaszczykm** [comment]
@cheld Yes, master is not red yet, because it uses cached dependencies. We have 5 integration tests to fix.

**Comment by maciaszczykm** [comment]
@Yamagami-Takuma @f-higashi Thank you!

**Pull request merged**